src/views/Home/index.view.tsx

```diff
@@ -1,12 +1,29 @@
 import * as React from "react";
 import Hero from "views/Home/components/Hero/index.view";
-import Footer from "./components/Footer/index.view";
 import Button from "components/Button/index.view";
 import EmailSubscriptionForm from "components/EmailSubscription/index.view";
 import MLHBanner from "components/MLHBanner/index.view";
 import Background from "components/Background/index.view";
 import Mission from "views/Home/components/Mission/index.view";
 import "./Home.scss";
+import MilestonesComponent from "./components/Milestones/index.view";
+import SponsorsComponent from "./components/Sponsors/index.view";
+import FooterComponent from "./components/Footer/index.view";
+
+// sponsor logo paths
+import Balsamiq from "images/sponsors/balsamiq.svg";
+import CIED from "images/sponsors/cied.svg";
+import CITRIS from "images/sponsors/citris.svg";
+import ChooseSantaCruz from "images/sponsors/chooseSantaCruz.svg";
+import Comcast from "images/sponsors/comcast.svg";
+import eBay from "images/sponsors/ebay.svg";
+import MLH from "images/sponsors/mlh.svg";
+import Pinnacle from "images/sponsors/pinnacle.svg";
+import SouthSwell from "images/sponsors/southSwell.svg";
+import Stickermule from "images/sponsors/stickermule.svg";
+import Stratovan from "images/sponsors/stratovan.svg";
+import Wolfram from "images/sponsors/wolfram.svg";
+import FifteenSeventeen from "images/sponsors/1517.svg";
```

Comment:
I think we can move this inside the `Sponsors` component

Reply:
I had the impression that the home index file would contain the props for our components since it's the view controller, and this already seems to be how data is serialized in our app. [I'd rather implement according to this established pattern](https://blog.andrewray.me/the-reactjs-controller-view-pattern/). 

If the increasing file size is your worry, we can modularize the props into JSON so data can be cleanly passed from the controller. But since we only have one day till January, I'd rather get this out the door than hyperfocus on cleaning up the codebase, especially since remaining optimizations don't affect functionality & we won't be developing this app extensively.